main.js

```diff
@@ -1,23 +1,41 @@
 var data = {
-  carDirection: 'east'
+  direction: 'east',
+  location: {
+    x: 0,
+    y: 0
+  },
+  currentInterval: 0
 };
 
 var $car = document.querySelector('.car');
 
 function turnCar(event) {
   if (event.key === 'ArrowDown') {
     $car.className = 'car south';
-    data.carDirection = 'south';
+    data.direction = 'south';
   } else if (event.key === 'ArrowUp') {
     $car.className = 'car north';
-    data.carDirection = 'north';
+    data.direction = 'north';
   } else if (event.key === 'ArrowRight') {
     $car.className = 'car east';
-    data.carDirection = 'east';
+    data.direction = 'east';
   } else if (event.key === 'ArrowLeft') {
     $car.className = 'car west';
-    data.carDirection = 'west';
+    data.direction = 'west';
+  }
+}
+
+function moveCarForward() {
+  data.location.x += 8;
+  $car.style.left = data.location.x + 'px';
+}
+
+function startCar(event) {
+  if (event.key === ' ') {
+    data.currentInterval = setInterval(moveCarForward, 16);
```

Comment:
Instead of making a separate function for this, you should move this to be a part of your existing `keydown` event handler.